http-server.ts

```diff
@@ -0,0 +1,35 @@
+import Koa = require('koa');
+import {router} from './router';
+import bodyParser = require('koa-bodyparser');
+import mongoose = require('mongoose');
+import {logger} from './logger';
+import nconf = require('nconf');
+
+nconf.file('./config.json');
```

Comment:
export to another file that initializing configuration + add option to override config from environment variables and cli arguments

Reply:
done